packages/framework/fluid-static/src/fluidContainer.ts

```diff
@@ -176,7 +176,7 @@ export interface IFluidContainer<TContainerSchema extends ContainerSchema = Cont
 	 * @remarks
 	 *
 	 * This should only be called when the container is in the
-	 * {@link @fluidframework/container-definitions#ConnectionState.Disconnected} state.
```

Comment:
`ConnectionState` is both a namespace and a type, so this link was ambiguous. The namespace contains the relevant documentation.